utils/tests/online.py

```diff
@@ -128,7 +128,7 @@ def test_purge_old(self):
         self.redis_conn.set('default_counter:2015-10', 'stuff2')
 
         tc.purge_old()
-        self.assertEqual(['default_counter:2015-10'],
+        self.assertEqual([b'default_counter:2015-10'],
```

Comment:
I still am unclear why changing this single unit test string fixes the issue. Given that this is a redis key result, I assume we don't need to sanitize this with `six.binary_type()`?